configs/gtp-emcts.cfg

```diff
@@ -0,0 +1,21 @@
+@include gtp-base.cfg
+
+# Over in emcts1/base.cfg we use the convention that the player at index 0
+# is the victim and the player at index 1 is the adversary. Here we use the
+# reverse convention. TODO: should we change this?
```

Comment:
Probably should change it to be consistent.

Reply:
Ok will do but this will also require a `KataGo-custom` PR